### PR TITLE
Fixed test apps

### PR DESCRIPTION
**Pull request by dtrabo**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Verify and fix build, updated swift

## Related PRs or issues

[AB#63002](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/63002)

### CODE REVIEW

File: SasquatchTV/SasquatchTV/ViewControllers/AnalyticsViewController.swift [hunk 1]
```diff
@@ -45,7 +45,7 @@ class AnalyticsViewController : UIViewController, UITableViewDataSource, AppCent
     table?.reloadData();
   }
 
-  func switchAnalyticsStatus(_ : Any) {
+    @objc func switchAnalyticsStatus(_ : Any) {
```
Comment (annakocheshkova) [inline SasquatchTV/SasquatchTV/ViewControllers/AnalyticsViewController.swift]:
```suggestion
  @objc func switchAnalyticsStatus(_ : Any) {
```

File: SasquatchTV/SasquatchTVSwift/AppDelegate.swift [hunk 2]
```diff
@@ -16,7 +16,8 @@ class AppDelegate : UIResponder, UIApplicationDelegate, MSCrashesDelegate {
 
     // Override point for customization after application launch.
     MSAppCenter.setLogLevel(MSLogLevel.verbose);
-    MSAppCenter.start("fbdeeff6-a549-4653-9f6d-f97dfe7f07b4", withServices : [MSAnalytics.self, MSCrashes.self]);
+    MSAppCenter.setLogUrl("https://in-integration.dev.avalanch.es")
+    MSAppCenter.start("e57f6975-9167-4b3b-b450-bbb87b717b82", withServices : [MSAnalytics.self, MSCrashes.self]);
```
Comment (MatkovIvan) [inline SasquatchTV/SasquatchTVSwift/AppDelegate.swift]: This is OK only as a temporary solution

Comment (guperrot) [reply SasquatchTV/SasquatchTVSwift/AppDelegate.swift]: What is the name of that app and why does it work only on INT?

Comment (clpolet) [reply SasquatchTV/SasquatchTVSwift/AppDelegate.swift]: Why is it a temporary solution. Is production not ready? Same for the other app.

Comment (clpolet) [reply SasquatchTV/SasquatchTVSwift/AppDelegate.swift]: This tvOS swift gets a 401 from ingestion, the app secret is probably wrong.

File: SasquatchTV/Config/Sasquatch Debug.xcconfig [hunk 3]
```diff
@@ -6,4 +6,5 @@ MTL_ENABLE_DEBUG_INFO = YES
 ENABLE_TESTABILITY = YES
 SWIFT_OPTIMIZATION_LEVEL = -Onone
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) DEBUG
-SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(inherited) DEBUG
\ No newline at end of file
+SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(inherited) DEBUG
+SWIFT_VERSION = 4.0
```
Comment (clpolet) [inline SasquatchTV/Config/Sasquatch Debug.xcconfig]: I don't think we need this one since inheriting from Sasquatch.xcconfig

**Pull request merged**